src/Core/Models/Api/Response/ProfileOrganizationResponseModel.cs

```diff
@@ -76,6 +82,9 @@ public ProfileOrganizationResponseModel(OrganizationUserOrganizationDetails orga
         public bool HasPublicAndPrivateKeys { get; set; }
         public string ProviderId { get; set; }
         public string ProviderName { get; set; }
+        public string FamilySponsorshipFriendlyName { get; set; }
```

Comment:
This is a weird name. What's a friendly name?

Reply:
It's used to help a user know what this sponsorship is related to for them. Since they could theoretically have multiple we wanted to give them a better way to differentiate.